mojo-data/mojos/rulefit/eyestate/v1.00/rulefit_v1.00_eyestate_scores2.txt

```diff
@@ -0,0 +1,40 @@
+[1.0, 0.47016793315368444, 0.5298320668463156]
+[1.0, 1.213029676705446E-12, 0.999999999998787]
+[1.0, 1.2045919817182948E-12, 0.9999999999987954]
+[1.0, 1.2214673716925972E-12, 0.9999999999987785]
+[1.0, 1.2214673716925972E-12, 0.9999999999987785]
+[1.0, 1.2299050666797484E-12, 0.9999999999987701]
+[1.0, 0.6210494923664125, 0.37895050763358756]
+[1.0, 1.2857093167895073E-9, 0.9999999987142907]
+[1.0, 1.2857093167895073E-9, 0.9999999987142907]
+[1.0, 1.2857093167895073E-9, 0.9999999987142907]
+[1.0, 1.2857093167895073E-9, 0.9999999987142907]
+[1.0, 1.2857093167895073E-9, 0.9999999987142907]
+[1.0, 1.2857093167895073E-9, 0.9999999987142907]
+[1.0, 1.2857093167895073E-9, 0.9999999987142907]
+[1.0, 1.2857093167895073E-9, 0.9999999987142907]
+[1.0, 1.2836183227449283E-9, 0.9999999987163817]
+[1.0, 1.280731520836298E-9, 0.9999999987192685]
+[1.0, 1.280731520836298E-9, 0.9999999987192685]
+[1.0, 9.384133470291545E-10, 0.9999999990615867]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+[0.0, 1.0, 0.0]
+java.lang.ArrayIndexOutOfBoundsException
```

Comment:
this exception is returned when testing predicting on vec too short (`mojoModel.score0(new double[]{0}, new double[]{0,0,0})`)